packages/compass/package.json

```
@@ -120,9 +120,11 @@
     "electron-rebuild": "electron-rebuild --only kerberos,keytar,interruptor",
     "prestart": "npm run electron-rebuild",
     "start": "npm run webpack serve -- --mode development",
-    "pretest": "npm run electron-rebuild",
-    "test": "xvfb-maybe electron-mocha --no-sandbox",
-    "posttest": "node ../../scripts/rebuild.js kerberos keytar interruptor",
+    "pretest-main": "npm run electron-rebuild",
+    "test": "npm run test-main && npm run test-renderer",
+    "test-main": "xvfb-maybe electron-mocha --no-sandbox \"./src/main/**/*.spec.*\"  \"./src/main/**/*.test.*\"",
+    "test-renderer": "xvfb-maybe electron-mocha --no-sandbox --config ./.mocharc.renderer.js \"./src/app/**/*.spec.js\"",
```

`no-sandbox` is already included in the config

```suggestion
    "test-renderer": "xvfb-maybe electron-mocha --config ./.mocharc.renderer.js \"./src/app/**/*.spec.js\"",
```